src/App.js

```diff
@@ -1,17 +1,30 @@
-import React, { Component } from 'react';
-import MyCalculator from './components/Calculator';
+import React from 'react';
+import { Route, Routes } from 'react-router-dom';
 
-export default class App extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {};
-  }
+// pages
+import Home from './pages/Home';
+import Quotes from './pages/Quote';
 
+// Components
+import Navbar from './components/Navbar';
+import MyCalculator from './components/Calculator';
+
+// eslint-disable-next-line react/prefer-stateless-function
+class App extends React.Component {
   render() {
     return (
-      <div className="App">
-        <MyCalculator />
-      </div>
+      <>
+        <div className="App">
+          <Navbar />
+          <Routes>
+            <Route path="/calculator" element={<MyCalculator />} />
+            <Route path="/" element={<Home />} />
+            <Route path="/quote" element={<Quotes />} />
+          </Routes>
```

Comment:
- [OPTIONAL] I think it would be nice to use the `WordSection` component from the `src/component/quote.js` file. Remember that the component displays several quotes from an API. It would be nice to use the `WordSection` component instead of the `Quotes` component which contains one static quote. 👍